types/ffmpeg-concat/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for ffmpeg-concat 1.1
+// Project: https://github.com/transitive-bullshit/ffmpeg-concat
+// Definitions by: Weslen Nascimento <https://github.com/weslenng>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Transition {
+    duration: number;
+    name: string;
+    params: any;
+}
+
+interface ConcatOptions {
+    audio?: string;
+    cleanupFrames?: boolean;
+    concurrency?: number;
+    frameFormat?: 'jpg' | 'png' | 'raw';
+    log?: (stdout: string) => void;
+    output: string;
+    tempDir?: string;
+    transition?: Transition;
+    transitions?: ReadonlyArray<Transition>;
+    videos: ReadonlyArray<string>;
+}
+
+declare function concat(concatOptions: ConcatOptions): Promise<void>;
+
+export default concat;
```

Comment:
I don't think this library exports a default. You're probably trying to do an `export =` syntax.